test/include/test_helpers.hpp

```diff
@@ -38,6 +38,7 @@ void TestCreateDirectory(string path);
 void TestDeleteFile(string path);
 void TestChangeDirectory(string path);
 string TestGetCurrentDirectory();
+string CreateTestDirectory(const string &path);
 string TestDirectoryPath();
```

Comment:
I think you removed the function `TestDirectoryPath()` no?

Reply:
`test_helpers.cpp:107` it's still there 👍